src/main/scala/cromwell/engine/backend/JobDescriptor.scala

```diff
@@ -0,0 +1,35 @@
+package cromwell.engine.backend
+
+import cromwell.engine.{AbortRegistrationFunction, WorkflowDescriptor}
+import cromwell.engine.workflow.{FinalCallKey, CallKey, BackendCallKey}
+import wdl4s._
+import wdl4s.values.WdlValue
+
+
+/** Aspires to be an equivalent of `TaskDescriptor` in the pluggable backends world, describing a job in a way
+  * that is complete enough for it to be executed on any backend and free of references to engine types.
+  * Currently a ways to go in freedom from engine types.
+  *
+  * @tparam K CallKey subtype
+  */
+sealed trait JobDescriptor[K <: CallKey] {
+  def workflowDescriptor: WorkflowDescriptor
+  def key: K
+  def locallyQualifiedInputs: CallInputs
+}
+
+final case class BackendCallJobDescriptor(workflowDescriptor: WorkflowDescriptor,
+                                          key: BackendCallKey,
+                                          locallyQualifiedInputs: CallInputs = Map.empty) extends JobDescriptor[BackendCallKey] {
+
+  // PBE temporarily still required.  Once we have call-scoped Backend actors they will know themselves and the
+  // backend won't need to be in the WorkflowDescriptor and this method won't need to exist.
+  def backend = workflowDescriptor.backend
+}
```

Comment:
Yes yes.. It was just a suggestion.This will anyways will be refactored later on :smile: